skops/card/tests/test_card.py

```diff
@@ -405,6 +428,69 @@ def test_with_metadata(self, card: Card, meth):
         assert result == expected
 
 
+class TestModelCardFromPath:
+    @pytest.mark.parametrize("suffix", [".pkl", ".pickle", ".skops"])
+    def test_model_card_str(self, suffix):
+        model0 = LinearRegression(n_jobs=123)
+        save_file_str = save_model_to_file(model0, suffix)
+        card_from_str = Card(save_file_str)
+        card_from_model0 = Card(model0)
+
+        assert card_from_model0.model.n_jobs == card_from_str.model.n_jobs
+
+    @pytest.mark.parametrize("suffix", [".pkl", ".pickle", ".skops"])
+    def test_model_card_path(self, suffix):
```

Comment:
I think this test could be unified with the previous one, as there is only one line difference, by parametrizing it over whether the `save_file` is converted to `Path`  or not.